src/fu-engine.c

```diff
@@ -2792,7 +2792,16 @@ fu_engine_md_refresh_device_verfmt (FuEngine *self, FuDevice *device, XbNode *co
 	/* found and different to existing */
 	if (verfmt != FWUPD_VERSION_FORMAT_UNKNOWN &&
 	    fu_device_get_version_format (device) != verfmt) {
+		FwupdRelease *rel = fu_device_get_release_default (device);
```

Comment:
do we only ever have one release?

Reply:
In a CAB file, I suppose it's technically possible that there are more than one release, and this could also be used by `fwupdmgr get-releases` I would guess.  So yes it probably needs to recognize all the releases then (or this needs to be plumbed deeper into the release itself not during the metadata refresh).

Reply:
applied to all releases